app/Models/Multiplayer/Room.php

```diff
@@ -343,7 +343,7 @@ public function startGame(User $owner, array $rawParams)
             'user_id' => $owner->getKey(),
         ]);
 
-        $this->setRelation('user', $owner);
+        $this->setRelation('host', $owner);
```

Comment:
renaming `$owner` and `assertUserRoomAllowance` to "host" seems useful

(and why is it called host when the column is `user_id` :expressionless: )